pkg/interceptors/cel/cel_test.go

```diff
@@ -0,0 +1,200 @@
+package cel
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"reflect"
+	"testing"
+
+	"github.com/google/cel-go/common/types"
+	"github.com/google/cel-go/common/types/ref"
+	"github.com/tektoncd/pipeline/pkg/logging"
+	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	fakekubeclient "knative.dev/pkg/client/injection/kube/client/fake"
+	rtesting "knative.dev/pkg/reconciler/testing"
+)
+
+// Allow configuration via a config map
+
+func TestInterceptor_ExecuteTrigger(t *testing.T) {
+	type args struct {
+		payload []byte
+	}
+	tests := []struct {
+		name    string
+		CEL     *triggersv1.CELInterceptor
```

Comment:
It looks like the only thing that changes here is the filter string. The test case struct would be simpler if we just use `filter string` here and create the CELInterceptor in the test.